### ISSUE TITLE
NTV2 Grid for Hesse (Germany)

### ISSUE THREAD

**Issue by SebastianKasan** [NaturalGIS/ntv2_transformations#48]
Please add this file for Hesse, Germany:

[HeTA2010.zip](https://github.com/NaturalGIS/naturalgis_ntv2_transformations/files/4664203/HeTA2010.zip)

**Comment by SebastianKasan** [comment]
It is the official transformation in gsa and gsb format including an explanatory pdf file...

**Comment by gioman** [comment]
@SebastianKasan Hi, thanks for the grid. What is the license of the gsb file? Does it allows re-distribution? 

At this moment we don't have time to add more transformations in out spare time, but if you need please feel free to submit a patch/PR or consider supporting the necessary work (2/3 hours of work).

Regards

**Comment by SebastianKasan** [comment]
Hi, this is free to use, the exact license is explained in the pdf but it says, any software company can use it as long as the name is attributed....

I will consider the patch thing. 
As I understand, QGIS can use NTV2 transformations also with the coordinate manager, is that correct?

**Comment by gioman** [comment]
> QGIS can use NTV2 transformations also with the coordinate manager

@SebastianKasan not sure if I understand this, can you elaborate?

**Comment by SebastianKasan** [comment]
Apparently you can use .gsb files in QGIS like so:
https://gis.stackexchange.com/questions/298011/how-to-use-gridfiles-gsb-in-qgis

**Comment by gioman** [comment]
> Apparently you can use .gsb files in QGIS like so:

@SebastianKasan What this plugins does is a slightly different thing:

QGIS can use ntv2 grids when you add layers in a project in order to reproject **on the fly** that layer using that specific transformation. The added layer is untouched, no changes or copies are made.

The goal of this plugin is to **create copies** of layers, where a ntv2 transformation is applied.

**Comment by SebastianKasan** [comment]
Ok, but when I save the layers that were reprojected on the fly, I can also set the "on the fly" projection  permanent, or not?

**Comment by gioman** [comment]
> Ok, but when I save the layers that were reprojected on the fly, I can also set the "on the fly" projection permanent, or not?

yes, but this is a Processing plugin and it allows you to do the save operation in **batch** mode, meaning that you get copies of many layers in just one operation (not manually one by one).

**Comment by gioman** [comment]
@SebastianKasan why have you closed this?
